app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  class CommentsController < ApplicationController
+    before_action :authenticate_user!
+
+    def index
+      post = Post.find(params[:post_id])
+      comments = post.comments
+      render json: comments
+    end
+
+    def create
+      post = Post.find(params[:post_id])
+      comment = post.comments.new(comment_params.merge(user: current_user))
+
+      if comment.save
+        render json: comment, status: :created
+      else
+        render json: comment.errors, status: :unprocessable_entity
+      end
+    end
+
```

Comment:
- [ ] I couldn't confirm the functionality of this method, I was unable to post comments to the api. I got the error message screenshot below. 👇 

<img width="608" alt="Untitled" src="https://github.com/chudisoft/RubyBlog/assets/103386183/75c2fe32-1edf-4824-b7de-0c5363aa82bf">

Reply:
Fixed.